com.unity.mobile.notifications/Runtime/iOS/Plugins/UnityNotificationManager.m

```diff
@@ -320,7 +321,7 @@ - (void)scheduleLocalNotification:(iOSNotificationData*)data
         CLLocationCoordinate2D center = CLLocationCoordinate2DMake(data->locationTriggerCenterX, data->locationTriggerCenterY);
 
         CLCircularRegion* region = [[CLCircularRegion alloc] initWithCenter: center
-                                    radius: data->locationTriggerRadius identifier: @"Headquarters"];
+                                    radius: data->locationTriggerRadius identifier: identifier];
```

Comment:
Why not create the NSString here? Would avoid allocation for other trigger types.

Reply:
It was already created and being used on line 339 and is required for other trigger types, I moved the creation up the class so we could access it here.